### ISSUE TITLE
Code for `Project` page is too big

### ISSUE THREAD

**Issue by nleroy917** [pepkit/pephub#298]
The code for the project page is just too big. Here are the imports alone:

```tsx
import { Sample } from '../../types';
import { StatusIcon } from '../components/badges/status-icons';
import { SchemaTag } from '../components/forms/components/shema-tag';
import { PageLayout } from '../components/layout/page-layout';
import { ProjectDataNav } from '../components/layout/project-data-nav';
import { Markdown } from '../components/markdown/render';
import { AddToPOPModal } from '../components/modals/add-to-pop';
import { DeletePEPModal } from '../components/modals/delete-pep';
import { EditMetaMetadataModal } from '../components/modals/edit-meta-metadata';
import { ForkPEPModal } from '../components/modals/fork-pep';
import { ProjectAPIEndpointsModal } from '../components/modals/project-api-endpoints';
import { ProjectPageheaderPlaceholder } from '../components/placeholders/project-page-header';
import { PopInterface } from '../components/pop/pop-interface';
import { ProjectConfigEditor } from '../components/project/project-config';
import { SampleTable } from '../components/tables/sample-table';
import { useAddStar } from '../hooks/mutations/useAddStar';
import { useConfigMutation } from '../hooks/mutations/useConfigMutation';
import { useRemoveStar } from '../hooks/mutations/useRemoveStar';
import { useTotalProjectChangeMutation } from '../hooks/mutations/useTotalProjectChangeMutation';
import { useNamespaceStars } from '../hooks/queries/useNamespaceStars';
import { useProjectAnnotation } from '../hooks/queries/useProjectAnnotation';
import { useProjectConfig } from '../hooks/queries/useProjectConfig';
import { useProjectViews } from '../hooks/queries/useProjectViews';
import { useSampleTable } from '../hooks/queries/useSampleTable';
import { useSubsampleTable } from '../hooks/queries/useSubsampleTable';
import { useValidation } from '../hooks/queries/useValidation';
import { useSession } from '../hooks/useSession';
import { dateStringToDate, dateStringToDateTime } from '../utils/dates';
import { copyToClipboard, getOS, numberWithCommas } from '../utils/etc';
import { canEdit } from '../utils/permissions';
import { downloadZip } from '../utils/project';
```

This is kind of ridiculous and will be a barrier to anyone coming into the codebase. It makes sense to me because I wrote it, but someone else will have a hard time navigating this. We should think about breaking this up.

**Comment by nsheff** [comment]
breaking it up more?

I was thinking actually grouping the components together more into modules and having files per module, instead of for every component...


also, `shema-tag` is misspelled...

**Comment by khoroshevskyi** [comment]
Fixed in pephub 0.11.9
